README.md

```diff
@@ -117,74 +131,54 @@ terraform apply
 * Now enter the `securityanalytics-taskexecution` directory to set up the ECS cluster for running scanning tasks on
 * Use terraform to initalise the infrastructure for this project:
 ```
-cd ../infrastructure
+cd infrastructure
 
 # you'll need to init whenever you add new providers in terraform
 # if prompted to migrate all workspaces to S3 then respond with 'yes'
-terraform init -backend-config "bucket=<your_bucket_name>-terraform-state"
+terraform init -backend-config "bucket=<app_name>-terraform-state"
 # select your workspace
 terraform workspace new <your_workspace>
 terraform apply
 ```  
 
-### Analytics Platform
-* Next the analytics platform needs to be deployed, enter the `securityanalytics-analyticsplatform` directory, and deploy the terraform. When deploying the infrastructure, elasticsearch takes around 10 minutes, so grab yourself a drink and wait...
+### Build steps for other projects
+
+For the rest of the steps, the build process is this same:
+* First set up your environment variable `PIPENV_VENV_IN_PROJECT` to `true`:
+  * Linux: `export PIPENV_VENV_IN_PROJECT=true`
+  * Powershell: `$Env:PIPENV_VENV_IN_PROJECT="true"`
+
 ```
 # update with the latest shared code first:
 git submodule init
 git submodule update --remote
 git submodule sync
 
-# setup pipenv
-# this works for Powershell, if you are using a different shell, set the environment variable accordingly:
-$Env:PIPENV_VENV_IN_PROJECT="true"
+# setup pipenv (make sure PIPENV_VENV_IN_PROJECT is set to true first)
 pipenv install --dev
+pipenv shell
 
 cd infrastructure
-
 # you'll need to init whenever you add new providers in terraform
-terraform init -backend-config "bucket=<your_bucket_name>-terraform-state"
+terraform init -backend-config "bucket=<app_name>-terraform-state"
 # select your workspace
 terraform workspace new <your_workspace>
 terraform apply
 ```
 
+### Analytics Platform
+* Next the analytics platform needs to be deployed, enter the `securityanalytics-analyticsplatform` directory, and follow the build steps above. When deploying the infrastructure, elasticsearch takes around 10 minutes, so grab yourself a drink and wait...
+
 ### Nmap scanner
-* This task requires some Python libraries to be installed first:
-```
-pip3 install boto3
-pip3 install requests_aws4auth
-```
+
 * Enter the `securityanalytics-nmapscanner` directory. If you have both Python 2 and Python 3 installed you might need to edit the `python` references in elastic_resources/index.tf to be `python3`. 
```

Comment:
Will this work if you only have python 3 installed? If so we should change the scripts

Reply:
it's a user thing if they have both installed (only works on python 3) - but if they have both then they'll have to figure out if they want to edit the script (easier) or change their default path (bit more complex depending on how their install is)